mstpd.spec

```diff
@@ -27,18 +32,21 @@ on Linux bridges when MSTP is used.
 ./autogen.sh
 %define _exec_prefix %{nil}
 %define _libexecdir /lib
-%configure
-make %{?_smp_mflags}
+%configure --with-systemdunitdir=%{_unitdir}
+%make_build
 
 %install
-rm -rf %{buildroot}
-make install DESTDIR=%{buildroot}
+%make_install
 
-sed -i -e 's|/etc/network/interfaces|/etc/sysconfig/network-scripts/bridge-stp|g' %{buildroot}/%{_libexecdir}/mstpctl-utils/ifquery
-sed -i -e 's|/etc/network/interfaces|/etc/sysconfig/network-scripts/bridge-stp|g' %{buildroot}/%{_libexecdir}/mstpctl-utils/mstp_config_bridge
+mkdir -p $RPM_BUILD_ROOT%{_prefix}/lib/NetworkManager/dispatcher.d
+install -m 755 -p utils/nm-dispatcher \
+  $RPM_BUILD_ROOT%{_prefix}/lib/NetworkManager/dispatcher.d/45-mstpd
```

Comment:
Shouldn't this be `%{buildroot}` instead of `$RPM_BUILD_ROOT`?

Reply:
They're the same, but RedHat packages seem to prefer $RPM_BUILD_ROOT in their install scripts...

Reply:
Hmmm... Yeah, now that you point it out, I see that they do seem to prefer that.  In that case, let's change the rest of the `%{buildroot}` macros we are currently using in this section to `$RPM_BUILD_ROOT`.